src/components/Filter/Filter.tsx

```diff
@@ -76,12 +78,31 @@ const Filter: React.FC<Props & TestProps> = props => {
         </button>
         {open && (
           <div css={optionsStyle({})(theme)} data-testid="filter-menu">
+            <button
+              css={{
```

Comment:
For consistency reasons, since we are using functions to pass style to the css prop, move this to the `Filter.styles` file.

Reply:
yes i changed thank you!